### PR TITLE
Add Poke plugin to create a workspace + invite first user

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This PR adds a global plugin in Poke so our customer team can create a workspace and invite a user. This has been asked a few times by @ahammour.

image

### CODE REVIEW

File: front/components/poke/PokeNavbar.tsx [hunk 1]
```diff
@@ -31,6 +31,9 @@ export const PokeNavbar: React.FC = () => (
         <Link href="/poke/templates">
           <div className="text-stucture-300 text-sm italic">Templates</div>
         </Link>
+        <Link href="/poke/plugins">
+          <div className="text-stucture-300 text-sm italic">Plugins</div>
```
Comment (JulesBelveze) [inline front/components/poke/PokeNavbar.tsx:35]: `nit`: Could use `Label` here

**Review by JulesBelveze — APPROVED** [review]
LGTM

File: front/components/poke/PokeNavbar.tsx (hunk 1 above)
Comment (flvndvd) [reply front/components/poke/PokeNavbar.tsx:35]: Our Sparkle component?

Comment (JulesBelveze) [reply front/components/poke/PokeNavbar.tsx:35]: yup

Comment (flvndvd) [reply front/components/poke/PokeNavbar.tsx]: Our `Label` component does not have any style when used in a hoverable state. Went with our `Button` in `ghost` variant.

**Pull request merged**